tests/Searchlight.Tests/LinqExecutorTests.cs

```diff
@@ -441,5 +441,30 @@ public void StringEqualsCaseInsensitive()
             Assert.IsNotNull(result);
             Assert.AreEqual(1, result.Count());
         }
+      
+        [TestMethod]
+        public void DefinedDateOperators()
+        {
+            var list = GetTestList();
+            
+            var syntax = src.Parse("hired < TODAY");
+
+            var result = syntax.QueryCollection(list);
+            
+            Assert.IsTrue(result.Any());
+            Assert.IsTrue(result.Count() == 3);
+
+            syntax = src.Parse("hired < TOMORROW");
+            result = syntax.QueryCollection(list);
+            
+            Assert.IsTrue(result.Any());
+            Assert.IsTrue(result.Count() == 4);
+            
+            syntax = src.Parse("hired > YESTERDAY");
+            result = syntax.QueryCollection(list);
+            
+            Assert.IsTrue(result.Any());
+            Assert.IsTrue(result.Count() == 4);
```

Comment:
Add tests:
* hired > 2020-01-01
* hired < 1985-01-01 (should return zero records)
* hired < tomorrow (lowercase)